When adacontrol can build with the ravenports base compiler
delete this patch.

--- build.gpr.orig	2019-10-30 08:19:42 UTC
+++ build.gpr
@@ -1,5 +1,4 @@
 with "asis";
-with "gnatcoll";
 project Build is
    for Languages use ("Ada");
    for Source_Dirs use ("src/");
